test/e2e/tests/test_hpo.py

```diff
@@ -18,40 +18,55 @@
 import logging
 from typing import Dict
 
-from e2e import service_marker, create_sagemaker_resource
-from e2e.replacement_values import REPLACEMENT_VALUES
-from acktest.resources import random_suffix_name
+from acktest.resources import random_suffix_name, load_resource_file
 from acktest.k8s import resource as k8s
+from e2e import (
+    resource_directory,
+    CRD_GROUP,
+    CRD_VERSION,
+    service_marker,
+    create_sagemaker_resource,
+    wait_for_status,
+)
+from e2e.replacement_values import REPLACEMENT_VALUES
+from e2e.bootstrap_resources import get_bootstrap_resources
+from time import sleep
 
 RESOURCE_PLURAL = "hyperparametertuningjobs"
-HPO_JOB_STATUS_CREATED = ("InProgress", "Completed")
-HPO_JOB_STATUS_STOPPED = ("Stopped", "Stopping")
 
+def _sagemaker_client():
```

Comment:
dont need this in test files, defined as a fixture in conftest.py in one of the earlier PRs

Reply:
I started to make this change and did too across two resources but using this one as a fixture and passing it around every method in the class seems unnecessary. Is there are a strong (performance or otherwise) reason for doing that ? (considering it is not a ton of code duplication. 
Another option could be that we could define it not as a fixture but an imported method which would make the code more readable.

Reply:
you are right, sounds good, can we move this to a common file rather than in each test file?

Reply:
done